pgmpy/models/BayesianNetwork.py

```diff
@@ -291,6 +299,13 @@ def get_cpds(self, node=None):
         ...                  ['intel', 'diff'], [2, 2])
         >>> student.add_cpds(cpd)
         >>> student.get_cpds()
+
```

Comment:
Could you please update the examples here such that we use one of the example models? If you search for `get_example_model`, you will find plenty of docstring examples using the `get_example_model` method to create a model.